machine/v1/types_controlplanemachineset.go

```diff
@@ -250,6 +251,10 @@ type FailureDomains struct {
 	// +optional
 	GCP *[]GCPFailureDomain `json:"gcp,omitempty"`
 
+	// Nutanix configures failure domain information for the Nutanix platform.
+	// +optional
+	Nutanix *[]configv1.NutanixFailureDomain `json:"nutanix,omitempty"`
```

Comment:
This doesn't need to be a pointer and the capitalisation should change to follow conventions
```suggestion
	// nutanix configures failure domain information for the Nutanix platform.
	// +optional
	Nutanix *[]configv1.NutanixFailureDomain `json:"nutanix,omitempty"`
```

Reply:
Fixed.